configs/plugins/groups/shared.yaml

```diff
@@ -0,0 +1,3 @@
+plugin_group:
```

Comment:
Why is this required, given the `common` and `runtime` fields in `action`?

Reply:
Yeah I agree that the current design makes it better to use `config["runtime"]` and `config["common"]`, and have been moving some of my Runner stuff to this, but it's not everywhere yet.

I have been thinking that we are still in-between design decisions for `common` and `runtime`. If we do move to the multi-repo design for both, it might make sense to continue treating them like normal Runner `Path`s / plugins (for build related actions). With that said, this file isn't strictly necessary.